### PR TITLE
Add script to update the password config.

### PR DESCRIPTION
**Pull request by sylmak**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,3 +30,29 @@ If you run into any errors, try logging into Firebase again:
...
+object with [these properties](https://firebase.google.com/docs/reference/admin/node/firebase-admin.auth.customstrengthoptionsconfig.md#customstrengthoptionsconfig_interface).
+
+Then, run:
+
+```bash
+firebase login
+npm run updatePasswordConfig
+```
+
+If you have deploy permissions, then you will have update password configuration
```
Comment (skeshive) [inline README.md:52]: can you add a section for how to get proper permissions?

File: update-password-config.cjs [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+app.initializeApp({ ...firebaseConfig, ...{ credential: accessToken } });
+
+auth
+  .getAuth()
+  .projectConfigManager()
+  .updateProjectConfig({
+    passwordPolicyConfig: {
+      enforcementState: 'ENFORCE',
+      forceUpgradeOnSignin: true,
```
Comment (skeshive) [inline update-password-config.cjs:36]: does this mean the user will have to reset their password if their new password doesn't match the updated config? if so, might be worth flagging this in the README so USL is aware

File: update-password-config.cjs
Comment (skeshive) [inline update-password-config.cjs:1]: do we need any additional steps in order for the auth app to start using the new password rules? or does firebase automatically handle this?

**Review by skeshive — APPROVED** [review]

File: update-password-config.cjs
Comment (sylmak) [reply update-password-config.cjs:1]: Firebase automatically handles this.

File: update-password-config.cjs (hunk 2 above)
Comment (sylmak) [reply update-password-config.cjs:36]: Yes. Added note in README.

File: README.md (hunk 1 above)
Comment (sylmak) [reply README.md:52]: It's the same as deploy - added a link.

**Pull request merged**